src/Audio_details.md

```diff
@@ -0,0 +1,192 @@
+These registers are not documented in any known Nintendo manual.
+
+### FF76 — PCM12 (CGB Mode only): Digital outputs 1 & 2 \[read-only\]
+
+The low nibble is a copy of sound channel 1's digital output, the high nibble a copy of sound channel 2's.
+
+### FF77 — PCM34 (CGB Mode only): Digital outputs 3 & 4 \[read-only\]
+
+Same, but with channels 3 and 4.
+
+## Finer technical explanation
+
+### DIV-APU
+
+A "DIV-APU" counter is increased every time `DIV`'s bit 4 (5 in [double-speed mode](<#FF4D — KEY1 (CGB Mode only): Prepare speed switch>)) goes from 1 to 0, therefore at a frequency of 512 Hz (regardless of whether double-speed is active).
+Thus, the counter can be made to increase faster by writing to `DIV` while its relevant bit is set (which clears `DIV`, and triggers the falling edge).
+
+Some events are tied to the DIV-APU counter:
+
+Event          | Every N ticks | Frequency[^div_apu_freq]
+---------------|---------------|-------------------------
+Envelope sweep | 8             | 64 Hz
+Sound length   | 2             | 256 Hz
+CH1 freq sweep | 4             | 128 Hz
+
+[^div_apu_freq]:
+Indicated values are under normal operation; the frequencies will obviously differ if writing to `DIV` to increase the counter faster.
+
+### Mixer
+
+A high-pass filter (HPF) removes constant biases over time.
+The HPFs therefore remove the DC offset created by inactive channels with an enabled DAC, and off-center waveforms.
+
+Enabling or disabling a DAC ([see below](#DACs)), adding or removing it using NR51, or changing the volume in NR50, will cause an audio pop.
+(All of these actions cause a change in DC offset, which is smoothed out by the HPFs over time, but still creates a pop.)
+
+The HPF is more aggressive on GBA than on GBC, which itself is more aggressive than on DMG.
+(The more "aggressive" a HPF, the faster it pulls the signal towards "analog 0"; this reduces the pops, but tends to also distort waveforms.)
+
+### DACs
+
+Channel <var>x</var>'s DAC is enabled if and only if `[NRx2] & $F8 != 0`; the exception is CH3, whose DAC is directly controlled by bit 7 of NR30 instead.
+Note that the envelope functionality changes the volume, but not the value stored in NRx2, and thus doesn't disable the DACs.
+
+If a DAC is enabled, the digital range $0 to $F is lineraly translated to the analog range -1 to 1, in arbitrary units.
+Importantly, the slope is negative: "digital 0" maps to "analog 1", not "analog -1".
+
+If a DAC is disabled, it fades to an analog value of 0.
+The nature of this fade is not entirely deterministic and varies between models.
+
+NR52's low 4 bits report whether the channels are turned on, not their DACs.
+
+### Channels
+
+A channel is activated by a write to NRx4's MSB, unless its DAC is off, which forces it to be disabled as well.
+The opposite is not true, however: a disabled channel outputs 0, which an enabled DAC will dutifully convert into "analog 1".
+
+A channel can be deactivated in one of the following ways:
```

Comment:
```suggestion
A channel is deactivated in the following cases:
```

Reply:
I'm trying to say that the list is exhaustive;